### DISCUSSION TITLE
Ask the user to remove conflict custom objects or automatically remove then.

### DISCUSSION THREAD

**Discussion by md500** [OpenRCT2/OpenRCT2#17431] [Feature requests]
### Operating System

Windows 10 Enterprise 1809 64bit

### OpenRCT2 build

OpenRCT2, v0.4.0 (c6302a1 on master) provided by GitHub

### Describe the issue

When i add custom scenarios,tracks and rides and objects, OpenRCT complains about object conflicts and user must remove the conflicting files manually from object folder to solve the problem

Will OpenRCT2 ask the user remove the conflicting files or remove file 1 conflicting with object file 2, or viceversa?

Thank you very much!

### Area(s) with issue?

Game opening

### Steps to reproduce

1. Download any fanmade scenarios like UCES (User Created Expansion Set), CPC (Creative Park Company) or AW (Adventure Worlds) and install to any directory. (to keep RCT1/RCT2 GoG files intact)
2. Copy the files from ObjData,Scenario,Tracks to corresponding OpenRCT folders.
3. Run the game with: `openrct.com --verbose with CMD`
4. Red text about: `Object conflict: '<Directory to OpenRCT User folder>\object\<any file>.DAT'` is displayed

### Attachments

![Silph](https://user-images.githubusercontent.com/3111925/174934625-5f12f773-d7d6-40ad-9297-8229782cd506.jpg)

**Comment by Gymnasiast** [comment]
This is **not** a bug, it’s a feature request and should have been created as such. (As the New Issue page clearly says.)
